torch/autograd/function.py

```diff
@@ -286,6 +286,31 @@ def apply(self, *args):
                 "of them."
             )
         user_fn = vjp_fn if vjp_fn is not Function.vjp else backward_fn
+
+        if self._is_compiled_autograd_tracing():
```

Comment:
We can't use the existing context, because it will have real tensors inside it -- not fake ones.  We need to lift the backwards method into a arg to the graph similar to how hooks are handled.